granulate_utils/metrics/yarn.py

```diff
@@ -6,8 +6,11 @@
 # Licensed under a 3-clause BSD style license (see LICENSE.bsd3).
 #
 import logging
+from functools import cached_property
 from typing import Dict, Iterable, List, Optional
 
+from packaging.version import Version
```

Comment:
`packaging` is not a builtin, it must be installed separately. Either add to requirements or do smth else.

Reply:
It's already there: https://github.com/Granulate/granulate-utils/blob/823c27ef6d87f9358e6b321da1919bb66daa5d4f/requirements.txt#L10